src/store/actions.ts

```diff
@@ -173,6 +174,7 @@ export default function createActions(
 			}
 			if ( params.spellVarCode !== undefined ) {
 				commit( SET_SPELLING_VARIANT, params.spellVarCode );
+				commit( SET_SPELLING_VARIANT_SEARCH_INPUT, params.spellVarCode );
```

Comment:
:thinking: Does this then in practice show "en" or "English (en)" in the SpellingVariant field?

Reply:
It only shows `en` (hence the note in the commit message). I’m thinking perhaps these search input manipulations should be done inside the component after all (back to `watchEffect()`?) – it doesn’t feel right for the store to use the language code provider and the message plugin to reconstruct the same search input value. But I think that can be done after this is merged.

Reply:
Mh, good point. But I agree that we best look at it in another PR